src/components/Sidebar/Sidebar.less

```diff
@@ -169,6 +169,12 @@
 			& > .lucid-Sidebar {
 				flex: 1;
 			}
+
+			// Addresses a Firefox specific rendering issue with a Submarine nested in a Sidebar.
+			& > .lucid-Submarine, x:-moz-any-link, x:default { 
```

Comment:
Also, the reason I needed this to be a browser specific selector is because applying these styles breaks rendering for the `PrimaryPane` in Safari, unfortunately.